tests/management/role/test_view.py

```diff
@@ -212,6 +212,30 @@ def test_create_role_allow_list_fail(self):
         response = self.create_role(role_name, in_access_data=access_data)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_create_role_appfilter_fields_fail(self):
```

Comment:
In addition to these, is there a test for creation of a role with invalid keys in the attributeFilter object?
```
access_data = [
            {
                "permission": "cost-management:*:*",
                "resourceDefinitions": [{"attributeFilter": {"One": "keyA", "Two": "boop", "Three": "valueA"}}],
            }
        ]
```

Reply:
Ah, I see. I missed the "foo" key in the test. This should be fine for what we need.